### ISSUE TITLE
Some way of having a screen identify itself

### ISSUE THREAD

**Issue by seanmtracey** [ftlabs/screens#56]
When looking at a screen that's already been assigned content, it's very hard to deduce what it's ID is without either popping the content it already has from the stack or adding something new. This issue is lessened by having nice semantically named screens, but in the case where the admin panel is just a sea of screen IDs, it would be nice to have a way to make the screen identify themselves without affecting the content stack it has.

Perhaps if we added a 'identify' option to the dropdown list in the footer? Maybe it could make the selected screen flash for a few ms (like when you make an error in Terminal and your volume is down way low) or it displays it's ID in the corner for a few seconds over the existing content.

**Comment by JakeChampion** [comment]
I think I may be misunderstanding. How can you trigger the screen to identify itself via the admin panel if you don't know the ID or name of the screen? Would you take a guess and ask a group of un-named screens to identify themselves in the hope that one of them is the screen that you are stood next to?

I think the use-case of wanting to know the ID/name of a screen that is nearby can be resolved via the physical web with fewer steps.

Apologies if I have misunderstood the scenario.

**Comment by seanmtracey** [comment]
No, you've got the scenario spot on. The physical web would be a great way to resolve, or narrow this problem, but with our current setup, we're still stuck with a trial and error system, but at least we could make figuring out the screen ID a less intrusive process.
